.github/workflows/golangci-lint.yml

```
@@ -0,0 +1,54 @@
+name: Run Quality Tests
```

```suggestion
name: Run Linters
```
Super pedantic: using `Quality` sounds to me as if linting ensures quality software. Which is I think is a bit more complicated than that...